homeworks/natalia.kokovina_NataliKokovina/homework_6/task1.js

```diff
@@ -0,0 +1,32 @@
+function addProtectedKey(firstKey, secondKey, user) {
+    let protectedVal;
+    Object.defineProperty(user, firstKey, {
+        get() {
+            return protectedVal;
+        },
+        set(value) {
+            protectedVal = value.toLowerCase().replace(/\b\w/g, l => l.toUpperCase());
+        },
+    });
+    Object.defineProperty(user, secondKey, {
+        get() {
+            return protectedVal;
+        },
+        set(value) {
+            if (value.charAt(0) === '+') {
+                protectedVal = `+${value.replace(/\D/g, '')}`;
+            } else {
+                protectedVal = value.replace(/\D/g, '');
```

Comment:
writing same code twice in 3 lines, not good...

Reply:
thanks, fixed